### PR TITLE
ref(templates): Remove unused global_sidebar

### PR DESCRIPTION
**Pull request by evanpurkhiser**
There are no templates that use this.

### CODE REVIEW

File: src/sentry/templates/sentry/layout.html [hunk 1]
```diff
@@ -56,36 +55,6 @@
...
-            propTypes: {
-              organization: PropTypes.any.isRequired
-            },
-            getChildContext: function () {
-              return {
-                organization: this.props.organization,
-              }
-            },
-            render: function () {
-              return React.createElement(SentryApp.OrganizationsLoader, {},
```
Comment (markstory) [inline src/sentry/templates/sentry/layout.html]: This might be the only usage of OrganizationsLoader left as well.

Comment (EvanPurkhiser) [reply src/sentry/templates/sentry/layout.html]: Nice it was 🙏

**Comment by EvanPurkhiser** [comment]
oops what have I done here

File: src/sentry/static/sentry/app/bootstrap.jsx [hunk 2]
```diff
@@ -154,10 +154,7 @@ globals.SentryApp = {
   ConfigStore: require('app/stores/configStore').default,
   Alerts: require('app/components/alerts').default,
   Indicators: require('app/components/indicators').default,
-  Sidebar: require('app/components/sidebar').default,
   SetupWizard: require('app/components/setupWizard').default,
-  OrganizationsLoader: require('app/components/organizations/organizationsLoader')
```
Comment (billyvg) [inline src/sentry/static/sentry/app/bootstrap.jsx]: Should we `rm` the component too?

File: src/sentry/templates/sentry/layout.html [hunk 3]
```diff
@@ -56,36 +55,6 @@
         {% show_system_status %}
     {% endif %}
 
-    {% block global_sidebar %}
-      {% if request.user.is_authenticated %}
-        <div id="blk_sidebar"></div>
-        <script>
-        $(function(){
-          var SidebarWrapper = createReactClass({
```
Comment (billyvg) [inline src/sentry/templates/sentry/layout.html]: Ooh I think we can get rid of https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/components/sidebar/index.jsx#L504 now too

**Review by billyvg — APPROVED** [review]

**Pull request merged**